<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGyro2MouseOptionDialog</class>
 <widget class="QDialog" name="QGyro2MouseOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gyro2Mouse Advanced Setting</string>
  </property>
  <widget class="QCheckBox" name="mouseXRevertCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>X Revert</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mouseXInputSourceComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="mouseXInputSourceLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X Source</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="mouseYInputSourceComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="mouseYRevertCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string>Y Revert</string>
   </property>
  </widget>
  <widget class="QLabel" name="mouseYInputSourceLabel">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Source</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
